### PR TITLE
WFDR-303-AboutUs

### PR DESCRIPTION
**Pull request by danielqyu**
## Summary of changes
Add aboutUs page

## Trello Task link
303-AboutUs

## Corresponding User Story Number and Title
10-About Us

## Detailed list of changes

Add aboutUs, access from ->footer on userlanding (about us)
Added static navbar when scrolling + small shadow

### CODE REVIEW

**Review by setaDM — CHANGES_REQUESTED** [review]
I don't think the about us page should have "meet the team" at the bottom where it just sends the user back to the same page. Maybe we can add a meet the team page and have a button in the bottom middle of the mission photo where it'll send users to that page. The meet the team page could have our names a small description and maybe a photo? (We can discuss during meeting tmrw). Also I think the text should be centralized underneath "traveling made better" to match the mission portion since the text there is centralized. Otherwise it looks good.

**Review by NickMoo5 — CHANGES_REQUESTED** [review]
I agree with David that the text under The big header "Travelling Made Better" should be centered.

Additionally the footers on all pages will need to be updated with this, currently only user landing has the link added to the footer. 

This will also need to be slightly updated after Peter's fluff page is merged as well.

Some points for further discussion during meeting is whether being logged in is required to view the webpage and potentially a navbar.

File: src/main/resources/templates/user/aboutUs.html [hunk 1]
```diff
@@ -102,127 +98,140 @@ <h4>WayFinder</h4>
...
+						<div class="row justify-content-center" style="padding-top: 50px">
+							<div class="col-md-5">
+								<i class="bi bi-tree"></i>
+								<h6>Draw people outdoors</h6>
+								<p style="font-size: smaller;">We believe humans are at their best when they're able to be outside.</p>
+							</div>
+							<div class="col-md-5">
+								<i class="bi bi-geo-alt"></i>
+								<h6>Inspire exploration</h6>
+								<p style="font-size: smaller;">We enourage people to explore the world around them.</p>
```
Comment (NickMoo5) [inline src/main/resources/templates/user/aboutUs.html]: spelling :))

File: src/main/resources/templates/user/aboutUs.html [hunk 2]
```diff
@@ -102,127 +98,140 @@ <h4>WayFinder</h4>
...
-			</div>
-		</div>
+<!--Our team-->
+<section>
+	<div>
+        <h3 style="text-align: center;">Our Team</h3>
+        <div class="row justify-content-center" style="padding-top: 20px">
+            <div class="col-md-6 text-center">
+                <img src="/images/profile/david.jpg" class="img-fluid rounded-circle" style="width: 120px; height: 120px; padding-bottom: 10px;">
+                <h6>David Muleh</h6>
```
Comment (NickMoo5) [inline src/main/resources/templates/user/aboutUs.html]: spelling :))

File: src/main/resources/templates/user/aboutUs.html [hunk 3]
```diff
@@ -102,127 +98,140 @@ <h4>WayFinder</h4>
...
-                                                <div class="form-icon-input form-fs-lg">
-                                                    <input type="text" class="form-control form-control-lg" id="location" name="location" placeholder="Enter a city" required>
-                                                    <span class="form-icon"><i class="bi bi-geo-alt fs-5"></i></span>
-                                                </div>
-												<div class="error-msg" id="locationErrorMsg"></div>
-                                            </div>
-											<div class="row p-4">
+            <div class="col-md-6 text-center">
+                <img src="/images/profile/nick.jpg" class="img-fluid rounded-circle" style="width: 120px; height: 120px; padding-bottom: 10px;">
+                <h6>Nicolas Mah</h6>
```
Comment (NickMoo5) [inline src/main/resources/templates/user/aboutUs.html]: spelling >:((((((( how could u im dissapointed

**Review by NickMoo5 — CHANGES_REQUESTED** [review]
dont forget to upload the imgs :))

**Review by setaDM — APPROVED** [review]
About us page and contact page looks good to me. The photo resolution is slightly distorted for some of the photos on the contact page, but it's not really a big deal and I also don't think you can really do anything about it since youre trying to fit it into the circle. If youre able to fix it then you can before you merge, but otherwise I think it looks good.

**Review by plee99 — APPROVED** [review]
the page looks awesome! I really like the design of the title, good work

**Pull request merged**